lib/iex/lib/iex/introspection.ex

```diff
@@ -521,10 +521,33 @@ defmodule IEx.Introspection do
               {format_callback(kind, fun, {fun, arity}, callbacks), doc}
           end)
 
+        docs =
```

Comment:
We need to move this to one specific clause inside `b/1`, otherwise we will show optional callbacks information even when doing something like `b GenServer.handle_call`.

Reply:
Okay! Will do it.

Reply:
I wonder if that's a bad thing. I think it would be nice to have `b GenServer.format_status` show that it's an optional callback.